api/src/utils/user-utils.js

```diff
@@ -37,7 +37,7 @@ const getEditableFields = (currentUser, memberId) => {
 };
 
 const filterViewableFields = (currentUser, member) => {
-  const viewable = getViewableFields(currentUser, member._id);
+  const viewable = getViewableFields(currentUser, member._id.toString());
```

Comment:
As a side note, this was causing a lot of issues. getViewableFields always expects the second param to be a string, but we were passing a mongoID object. This caused the backend to say that a user could view their own "level" and "areDuesPaid", but it would never return these fields because filterViewableFields was calling getViewableFields with the wrong data type.

Reply:
Good catch!